gulpfile.js

```diff
@@ -44,6 +45,22 @@ const lint = () => gulp.src(['src/**/*.js', '*.js'])
   .pipe(eslint.format())
   .pipe(eslint.failAfterError());
 
+function isFixed(file) {
+  return file.eslint != null && file.eslint.fixed;
+}
+
+/**
+ * @summary Use ESLint linting *.js files besides source files in node_modules and fix
+ * some simple errors.
+ *
+ * @returns {Stream}
+ */
+const lintfix = () => gulp.src(['**/*.js', '!node_modules/**'])
```

Comment:
Since our directory structure changed recently (many files moved to the `src/` directory), could you update these globs to be the same as the `lint` task above? I think we should refactor the glob array into a variable and use it in both the `lint` and `lintfix` tasks to make sure they are linting the same files.

Also, eslint ignores the `node_modules/` directory by default: https://eslint.org/docs/user-guide/configuring#eslintignore so you shouldn't need `!node_modules/**` when working with eslint.